### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by kajgrant**
Updated the readme.md docs.
I'm wondering if you would like me to add:

- An acknowledgements section

  and/or

- A deployments steps section (i.e. building static assets and how to deploy to AWS)

Referencing issue #157

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,58 @@
...
+## Installation
+
+1. Install Node modules in the top directory
+
+```
+cd StrikeOrSike
+npm install
+```
+
+2. Install Node modules in the client directory
```
Comment (ConorMurphy21) [inline README.md:28]: Shouldn't be necessary, the top level install should do everything for you

File: README.md [hunk 2]
```diff
@@ -1,5 +1,58 @@
...
+cd ../server
+npm install
+```
+
+## Running the client/sever
+
+1. Start the server
+
+```
+npm run dev
```
Comment (ConorMurphy21) [inline README.md:47]: If you do it in webstorm you can run in debug mode, and also not have to worry about using 2 shells

**Review by ConorMurphy21 — CHANGES_REQUESTED** [review]
Sorry, I refactored a lot of code over Christmas so the setup is slightly different from when we last discussed. If you want to add Acknowledgements that would be cool. I'll add build instructions later when I've finished refactoring.

File: README.md (hunk 1 above)
Comment (kajgrant) [reply README.md:28]: Oh nice ok, will change

File: README.md (hunk 2 above)
Comment (kajgrant) [reply README.md:47]: General convention should be not reliant on an IDE, no? I'm not using webstorm, but I can add in a note specifically for webstorm if you would like.

Comment (ConorMurphy21) [reply README.md:47]: Yeah def can leave the commands in I just thought you should know cuz it'll be easier for you

Comment (kajgrant) [reply README.md:47]: Would you like me to add a small section for webstorm or just leave it as is?

**Review by ConorMurphy21 — APPROVED** [review]
looks good. shows all of the important steps.

**Pull request merged**